### PR TITLE
Remove Tier 2 components from odh-manifests

### LINKED ISSUE #577: Migration of ODH Contrib Tier 2 components to new organization
As we move to re-architect ODH and its components, we want to provide clear guidance on the support level of components that will be part of the ODH Core (Tier 0) offering with full long term support vs ODH Contrib (Tier 2) components that are solely maintained by the designated maintainer with no limited support.

New GitHub organization for ODH Contrib (Tier 2) - https://github.com/opendatahub-io-contrib
- [x] #586
- [ ] Move `odhpachyderm` to ODH Tier 2 organization
- [ ] Move `trino` to ODH Tier 2 organization
- [ ] Move `superset` to ODH Tier 2 organization
- [ ] Move `openshift-service-mesh` to ODH Tier2 organization
- [ ] Move `odhargo` to ODH Tier 2 organization
- [ ] Move `odhseldon` to ODH Tier 2 organization
- [ ] Move `openshift-pipelines` to ODH Tier 2 organization
- [ ] Move `kafka` to ODH Tier 2 organization
- [x] #640
- [ ] Move `radanalyticsio` to ODH Tier 2 organization
- [ ] Move `ceph-nano` to ODH Tier 2 organization
- [ ] Move `hue` to ODH Tier 2 organization
- [ ] Move `thriftserver` to ODH Tier 2 organization

Below is the list of components provided in currently availaible in odh-manifests and where we expect them to exists once the ODH Re-architecture is complete.  Ti The initial migration will still support the deployment of these components using ODH Operator `kfdef` custom resources

ODH Core (TIER 0)
===
- odh-dashboard
- jupyterhub
- odh-notebook-controller

ODH Incubating (TIER 1)
===
- model-mesh
- grafana
- prometheus

ODH Contrib (TIER 2) - github.com/opendatahub-io-contrib
===
- odhpachyderm
- google-spark-operator
- airflow helm chart
- trino
- superset
- openshift-service-mesh
- odhargo

ODH Contrib (TIER 2 - DEPRECATED) - github.com/opendatahub-io-contrib
===
- Provided by OperatorHub -  odhseldon, openshift-pipelines, kafka
- ai-library
- radanalyticsio
- ceph-nano - Possible replacement by noobaa-operator
- hue
- thriftserver

### LINKED ISSUE #640: Move `ai-library` to ODH Tier 2 organization

### PR DESCRIPTION
**Pull request by LaVLaS**
Removes all of the components that will be migrated to the [opendatahub-io-contrib/odh-contrib-manifests](https://github.com/opendatahub-io-contrib/odh-contrib-manifests) repo

Closes #640, Closes #577 

Signed-off-by: Landon LaSmith <LLaSmith@redhat.com>

## How Has This Been Tested?
Deploy ODH Core components and verify there is no change in functionality

## Merge criteria:
CI passes with no errors to ODH Core components

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,39 +4,33 @@ A repository for [Open Data Hub](https://opendatahub.io) components Kustomize ma
...
-* Documentation: https://opendatahub.io/docs.html (applicable for version 0.5.1)
+* Documentation: https://opendatahub.io/docs.html
 * Mailing lists: https://opendatahub.io/community.html
 * Community meetings: https://gitlab.com/opendatahub/opendatahub-community
 
-## Components
+## ODH Core Components
 
-Open Data Hub is an end-to-end AI/ML platform on top of OpenShift Container Platform which provides various tools for Data Scientists and Engineers. The components currently available are:
+Open Data Hub is an end-to-end AI/ML platform on top of OpenShift Container Platform that provides a core set of integrated components to support end end-to-end MLOps workflow for Data Scientists and Engineers. The components currently available as part of the ODH Core deployment are:
```
Comment (anishasthana) [inline README.md:13]:
```suggestion
Open Data Hub is an end-to-end AI/ML platform on top of OpenShift Container Platform that provides a core set of integrated components to support the entire MLOps workflow for Data Scientists and Engineers. The components currently available as part of the ODH Core deployment are:
```

File: README.md [hunk 2]
```diff
@@ -4,39 +4,33 @@ A repository for [Open Data Hub](https://opendatahub.io) components Kustomize ma
...
-* [Pachyderm](odhpachyderm/README.md)
-* Data Catalog
-    * [Hue](hue/README.md)
-    * [Thrift Server](thriftserver/README.md)
-* [Trino](trino/README.md)
+* [ODH Dashboard](odh-dashboard/README.md)
 * [ODH Notebook Controller](odh-notebook-controller/README.md)
 * [Data Science Pipelines](data-science-pipelines/README.md)
+* [ModelMesh](model-mesh/README.md)
+* [Grafana](grafana/README.md) & [Prometheus](prometheus/README.md)
```
Comment (anishasthana) [inline README.md:19]: We're keeping Prometheus as a tier zero component? I thought there were discussions around driving users towards UWM.

**Review by anishasthana — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (LaVLaS) [reply README.md:19]: Do you know if the current prometheus component in ODH is still required for any monitoring workloads?

Comment (anishasthana) [reply README.md:19]: Not off the top of my head. I know for Modelmesh we say use User Workload Monitoring.

Comment (LaVLaS) [reply README.md]: Thx for the reminder.  I removed it from the README since we do have plans to remove it

**Review by anishasthana — COMMENTED** [review]
/lgtm

**Pull request merged**